src/writer/to_code.rs

```diff
@@ -137,7 +137,7 @@ impl ToCode for Chunk {
                 );
                 let self_str = if in_trait { "Self, " } else { "" };
                 let s2 = format!(
-                    "\tSome(*(&{}::<{}F> as *const _ as *const _)), Box_::into_raw(f))",
+                    "\tSome(transmute({}::<{}F> as usize)), Box_::into_raw(f))",
```

Comment:
For completeness, can we do

```suggestion
                    "\tSome(transmute::<_, unsafe extern "C" fn()>::({}::<{}F> as *const ())), Box_::into_raw(f))",
```

That's less likely to fall apart and closer to the unsafe code guidelines.

Reply:
Ok, but without second `::`

Reply:
Indeed, updated comment

Reply:
```
Some(transmute::<_, unsafe extern "C" fn()>(
                    activate_trampoline::<Self, F> as *const (),
                )),
```

Reply:
Looks correct

Reply:
Only problem of this change that we need change manual code again instead just reverting commit :(

Reply:
Need to do that anyway, there were some other changes in the manual code that should be kept

Reply:
@EPashkin Can you update?

Reply:
Oops, forgot to push.
Done